src/include/miopen/solver/implicitgemm_util.hpp

```diff
@@ -212,7 +212,8 @@ static inline bool IsXdlopsSupport(const ExecutionContext& ctx)
     // 1) inline asm may crash
     // 2) llvm intrin may has incorrect results
     const bool is_xdlops_supported = StartsWith(ctx.GetStream().GetDeviceName(), "gfx908") ||
-                                     StartsWith(ctx.GetStream().GetDeviceName(), "gfx90a");
+                                     StartsWith(ctx.GetStream().GetDeviceName(), "gfx90a") ||
+                                     StartsWith(ctx.GetStream().GetDeviceName(), "gfx940");
```

Comment:
[Recommendation] It may worth doing this:
```suggestion
                                     StartsWith(ctx.GetStream().GetDeviceName(), "gfx94");
```
in order to support the future steppings of MI300.

Reply:
👍